scripts/mkcloud

```diff
@@ -250,6 +250,10 @@ function sshrun
         export scenario=$scenario ;
         export host_mtu=$host_mtu ;
         export architectures='$architectures';
+        export cloud6_iso_path=$cloud6_iso_path ;
```

Comment:
these are candidates for the `want_` variables
just name them `want_iso_xxx` and they will be passed to the admin node via magic